fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+	font-family: 'Gotham Pro';
+	src: url('fonts/GothamProBold.eot');
+	src: local('☺'), url('fonts/GothamProBold.woff') format('woff'), url('fonts/GothamProBold.ttf') format('truetype'), url('fonts/GothamProBold.svg') format('svg');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)